ingest/log/logging.go

```diff
@@ -394,7 +395,7 @@ func (l *Logger) Debug(msg string, sds ...rfc5424.SDParam) error {
 	return l.outputStructured(DEFAULT_DEPTH, DEBUG, msg, sds...)
 }
 
-// Infof writes an INFO level log to the underlying writer using a format string,
+// Info writes an INFO level log to the underlying writer using a format string,
```

Comment:
```suggestion
// Info writes an INFO level log to the underlying writer using a static string,
```
I think this is supposed to be static string here.